src/bin/test_account_add.sh

```diff
@@ -12,4 +12,10 @@ if [[ -z "$php" ]]; then
   exit 1
 fi
 
-kubectl -n "${HELM_NAMESPACE}" exec -it "$php" -- bash -c "wp user create p4_test_user p4test+user@greenpeace.org --role=editor --user_pass=${WP_TEST_USER}"
+# Set kubernetes command with namespace
+kc="kubectl -n ${HELM_NAMESPACE}"
+
+if ! $kc exec "$php" -- wp user create p4_test_user p4test+user@planet4.test --user_pass="${WP_TEST_USER}" --role=editor; then
```

Comment:
> greenpeace.org domain would force a google login switching to local domain

We are actually [excluding](https://github.com/greenpeace/planet4-master-theme/blob/cd243159d3ae0f12523b3241c3730e95edacc183/src/MasterSite.php#L993-L996) users with a `+` sign from going through Google, because of [this](https://planet4.greenpeace.org/manage/administer/profile/#editing-an-avatar-for-someone-else) :grin: But it's fine to do it with the local domain instead :+1: